app/views/spree/admin/products/new.html.haml

```diff
@@ -22,20 +21,22 @@
             = f.error_message_on :name
       .sixteen.columns.alpha
         .eight.columns.alpha
-          = f.field_container :units do
+          = f.field_container :variant_unit do
```

Comment:
From this https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/helpers/spree/admin/base_helper.rb#L9

I understand that the field container name must match the name given to  `f.error_message_on`